erpnext/accounts/report/cash_flow/cash_flow.py

```diff
@@ -135,14 +135,21 @@ def get_cash_flow_accounts():
 		"section_name": "Investing",
 		"section_footer": _("Net Cash from Investing"),
 		"section_header": _("Cash Flow from Investing"),
-		"account_types": [{"account_type": "Fixed Asset", "label": _("Net Change in Fixed Asset")}],
+		"account_types": [
+			{"account_type": "Current Asset", "label": _("Net Change in Current Assets")},
+			{"account_type": "Fixed Asset", "label": _("Net Change in Fixed Assets")},
+		],
 	}
 
 	financing_accounts = {
 		"section_name": "Financing",
 		"section_footer": _("Net Cash from Financing"),
 		"section_header": _("Cash Flow from Financing"),
-		"account_types": [{"account_type": "Equity", "label": _("Net Change in Equity")}],
+		"account_types": [
+			{"account_type": "Current Liability", "label": _("Net Change in Current Liabilities")},
+			{"account_type": "Liability", "label": _("Net Change in Liabilities")},
```

Comment:
This account type is ambiguous and may not necessarily be used for long term liabilities.

Reply:
Agreed, we should rename and precisely define it alongside the proposed canon of:
- Current Assets/Liabilities 
- Current Operating A/L
- Long Term Liabilities

In order to achieve precise disambiguation, I propose to implement a suitable help message on the account doctype which clearly defines each account type in a way that's easily discoverable by the user (i.e. pop-up) and explains the impact and consequences of classifying an account into any of those types.